android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -117,6 +117,7 @@ public void setSubtitleStyle(SubtitleStyle style) {
             subtitleLayout.setFixedTextSize(TypedValue.COMPLEX_UNIT_SP, style.getFontSize());
         }
         subtitleLayout.setPadding(style.getPaddingLeft(), style.getPaddingTop(), style.getPaddingRight(), style.getPaddingBottom());
+        subtitleLayout.setVisibility(style.getOpacity() == 0 ? View.GONE : View.VISIBLE);
```

Comment:
I think you should be able to use  
subtitleLayout.setAlpha(style.getOpacity()); if opacity is != 0
It would be more flexible

Reply:
done :) implemented as follows:

```java
if (style.getOpacity() != 0) {
    subtitleLayout.setAlpha(style.getOpacity());
} else {
    subtitleLayout.setVisibility(View.GONE);
}
```